dart/lib/src/browser_client.dart

```diff
@@ -4,14 +4,33 @@
 
 /// A pure Dart client for Sentry.io crash reporting.
 import 'dart:convert';
-import 'dart:html' hide Event, Client;
+import 'dart:html' show window;
```

Comment:
@rxlabz I am a bit lost to understand what `show window` means, help :)

Reply:
it's a way to tell to the compiler that we only need the `window` member off this package :)